partner.md

```diff
@@ -1,47 +1,103 @@
+
+  These can seem a little vague, especially to non-technically oriented people, but if you are having difficulty sharing, storing or accessing data for your organization, we might be able to help.
+  We can and have built other kinds of software and this is only to give you an idea. We'll definitely find a way to help!
+</div>
+
+<div class="col-md-6" markdown="1">
+  <div class="center">
+    <h2><div class="highlight icon icon-ecology-09"></div></h2>
+    <h5 class="center">Problems we solve</h5>
+  </div>
+  1. Are you recording data with pen/paper and later transferring that data to a computer?
+  2. Do you pass word/excel/text files around with collaborators to share information?
+  3. Are you using software for a project now that wasn't built for the kind of work you are doing (“making it work” even though it’s super painful)?
+  4. Do you take data entered somewhere, manually manipulate it and send it on elsewhere?
+  5. Do you have databases that require different access levels? For example, a volunteer can record and enter data, a principal researcher can then lock it and prevent further volunteer edits and only allow researchers to edit to protect data integrity, reviewers can see but not edit data, and so on.
+  6. Do you use desktop programs to record information so you are locked into doing it from a certain location but it would be more convenient to have it web based so you can see/edit from anywhere?
+</div>
+
+<div class="col-md-6" markdown="1">
+  <hr>
+  <div class="center">
+    <h2><div class="highlight icon icon-seo-icons-38"></div></h2>
+    <h5 class="center">Cost</h5>
+  </div>
+  The project development is free! We do this to help the people who are making the world a better place. For the past three years we have also been very fortunate to have received sponsorships from hosting companies to host the projects.
+
+  However, there is no guarantee these sponsorships will continue. You may need to find hosting for the project once it is built. We will work with you to find the most affordable solution for your project or to integrate it with your current hosting.
+</div>
+
+<div class="col-md-6" markdown="1">
+  <hr>
+  <div class="center">
+    <h2><div class="highlight icon icon-faces-users-04"></div></h2>
+    <h5 class="center">Your involvement</h5>
+  </div>
+  In order to build you the best software possible we need your participation so we can understand your needs. We require you to commit to at least two meetings with the person leading your team in the lead up to the event.
+
+  You will also need to be available at least once during the event, ideally each day. Your team will coordinate the times. This is necessary because the team building your software will inevitably run into design decisions that will require your input in order to resolve and keep the project moving forward.
+</div>
+
+<div class="col-md-4" markdown="1">
```

Comment:
I get the flow for moving this lower, and at the same time kind of like having it up higher so people who prefer video to text can quickly see it.  Ultimately your call!

Reply:
either works!